mtags/src/main/scala/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -16,7 +16,8 @@ import org.eclipse.{lsp4j => l}
 
 class CompletionProvider(
     val compiler: MetalsGlobal,
-    params: OffsetParams
+    params: OffsetParams,
+    clientSupportsSnippets: Boolean
```

Comment:
FYI, `compiler.metalsConfig` already has this info

Reply:
ah, makes sense! I even thought about it, but then I forgot 😄 I’ll fix it